arrow/src/main/resources/arrow-versions-config.json

```diff
@@ -38,16 +38,18 @@
       "1.3.41",
       "1.3.50",
       "1.3.61"
+    ]
+  },
+  {
+    "version": "0.11.0",
+    "supportedKotlinVersions": [
+      "1.4.0"
     ],
     "latestStable": true
   },
   {
-    "version": "0.11.0-SNAPSHOT",
+    "version": "1.0.0-SNAPSHOT",
```

Comment:
Are we moving into 1.0.0? I thought we were releasing another one on top of 0.11.0 for the leftovers

Reply:
Yes, 1.0.0 is the next scheduled version